### ISSUE TITLE
The script doesn't work with Alpine 3.14-based images

### ISSUE THREAD

**Issue by ad-on-is** [mlocati/docker-php-extension-installer#362]
I have setup an automated PHP image build for web development with extensions. Today I pulled one of my automated built images (PHP 7.4.20) on my dev setup and encountered that `mysqli` was not working. After further inspection, I found out that none of the extensions were installed.

This is `phpinfo()` on my dev-setup
Notice how "Additional .ini files parsed" only contains sodium.

None of these work: `php:7.3-fpm-alpine`, `php:7.4-fpm-alpine`, `php:8.0-fpm-alpine`

![image](https://user-images.githubusercontent.com/9553253/123828583-ef6c9f00-d901-11eb-901f-c9b1c9dc3eff.png)

This is `phpinfo()` on my production environment, which works just fine.

![image](https://user-images.githubusercontent.com/9553253/123828407-bc2a1000-d901-11eb-9eb6-03e0d4fb736b.png)

This is the code of my outomated build

```bash

sync && install-php-extensions --cleanup \
    apcu \
    bcmath \
    bz2 \
    calendar \
    exif \
    gd \
    imagick \
    intl \
    mcrypt \
    memcached \
    mysqli \
    opcache \
    pdo_mysql \
    pdo_pgsql \
    pgsql \
    redis \
    soap \
    uuid \
    zip \
    xdebug
```

**Comment by mlocati** [comment]
It seems they switched to Alpine 3.14:

```sh
docker run --rm -it php:7.4-fpm-alpine cat /etc/os-release
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.14.0
PRETTY_NAME="Alpine Linux v3.14"
HOME_URL="https://alpinelinux.org/"
BUG_REPORT_URL="https://bugs.alpinelinux.org/"
```

See also https://github.com/mlocati/docker-php-extension-installer/pull/353

For the time being, you can use `php:7.3-fpm-alpine3.13`, `php:7.4-fpm-alpine3.13`, `php:8.0-fpm-alpine3.13`

**Comment by duyleekun** [comment]
Change to php:7.4-fpm-alpine3.13
Everything docker images just bumped to 3.14

**Comment by ad-on-is** [comment]
Ah, thank you guys! It's working now.

**Comment by mlocati** [comment]
Let's keep this open until the script will support alpine 3.14

**Comment by ad-on-is** [comment]
Oh ok, I thought because of #353 this issue might be redundant.

**Comment by mlocati** [comment]
Fixed by #364
